src/index.js

```diff
@@ -0,0 +1,27 @@
+import './style.css';
+
+document.addEventListener('DOMContentLoaded', () => {
+  const todoItems = [
+    {
+      description: 'Do homework',
+      completed: 'true',
+      index: 0,
+    },
+    {
+      description: 'Go shopping',
+      completed: 'true',
+      index: 1,
+    },
+    {
+      description: 'Prepare a budget',
+      completed: 'true',
+      index: 2,
+    }];
+  todoItems.forEach((todoItems) => {
+    const display = document.querySelector('.display');
+    const items = document.createElement('li');
+    const textItems = document.createTextNode(todoItems.description);
+    items.appendChild(textItems);
+    display.appendChild(items);
+  });
```

Comment:
- [ ] You still need to style the HTML output to match the design shared with you.

  Your Implementation | Required Design
  :---:|:---:
  ![todo-youtube-2](https://user-images.githubusercontent.com/54848363/177589297-277c4259-2575-4606-9f61-5ba1e488d31c.jpg) | ![todo-youtube](https://user-images.githubusercontent.com/54848363/177589307-79d7a74f-338f-497d-b50a-3b760ee05cf2.jpg)

Reply:
Thanks for the feedback

Reply:
I was very confused, but now i feel I have gotten some direction